src/pushsource/_impl/backend/koji_source.py

```diff
@@ -642,12 +642,14 @@ def _get_operator_item(self, nvr, meta, archives):
                 'archive "%s"'
             ) % (nvr, archive_name)
             raise ValueError(message)
+        helper = ContainerArchiveHelper(meta, operator_archive[0])
```

Comment:
I'm not confident that this works.

- 100% of the testcases show an empty 'labels' added to the generated items.
- `operator_archive[0]` here is an operator_manifests.zip archive.  ContainerArchiveHelper.labels works by reading out from the `extra` field on the archive.  When I look at an example, like https://brewweb.engineering.redhat.com/brew/archiveinfo?archiveID=8947693, there are no labels in the `extra` field of an operator_manifests.zip.  So I'm not sure what you expect this to return.

Can you please either update testcases so there is at least one case with realistic input data and non-empty output?

Also, is there any JIRA issue I can read for improved understanding of what this field is supposed to be used for?

Reply:
yeah, you're right. For some reason I thought it's output of filter which produces the container archive, not operator_manifests.zip